src/pk.c

```diff
@@ -11916,6 +11916,37 @@ int wolfSSL_EC_KEY_LoadDer_ex(WOLFSSL_EC_KEY* key, const unsigned char* derBuf,
         else {
             ret = wc_EccPublicKeyDecode(derBuf, &idx, (ecc_key*)key->internal,
                 derSz);
+            if (ret < 0) {
+                ecc_key *tmp = (ecc_key*)XMALLOC(sizeof(ecc_key),
+                             ((ecc_key*)key->internal)->heap, DYNAMIC_TYPE_ECC);
+                if (tmp == NULL) {
+                    ret = -1;
+                }
+                else {
+                    /* We now try again as x.963 [point type][x][opt y]. */
+                    ret = wc_ecc_init_ex(tmp, ((ecc_key*)key->internal)->heap,
+                                         INVALID_DEVID);
+                    if (ret == 0) {
+                        ret = wc_ecc_import_x963(derBuf, derSz, tmp);
+                        idx += derSz;
```

Comment:
I think `idx` should not be considered valid or guaranteed initialized to zero. Recommend `idx = derSz`.